playground/keycloak/Keycloak.ApiService/Program.cs

```diff
@@ -6,9 +6,10 @@
 // Add services to the container.
 builder.Services.AddProblemDetails();
 
-builder.AddKeycloakJwtBearer("keycloak", configureJwtBearerOptions: options =>
+builder.AddKeycloakJwtBearer(configureJwtBearerOptions: options =>
```

Comment:
I think the resource name argument is still required to ensure you use the right URL, e.g. if this app references multiple Keycloak resource instances.

Reply:
Done!